pystiche_papers/ulyanov_et_al_2016/_modules.py

```diff
@@ -150,6 +217,25 @@ def conv_block(
 
 
 class JoinBlock(nn.Module):
+    r"""JoinBlock from :cite:`ULVL2016`.
+
+    This block cats an arbitrary number of inputs along the ``channel _dim`` with
```

Comment:
I only used `cat` to shorten my comment :smile:

```suggestion
    This block concatenates an arbitrary number of inputs along the ``channel _dim`` with
```